sdk/src/main/scala/com/horizen/account/sc2sc/AbstractCrossChainRedeemMessageProcessor.scala

```diff
@@ -86,14 +86,18 @@ abstract class AbstractCrossChainRedeemMessageProcessor(
     }
   }
 
+  override def addScTxCommitmentTreeRootHash(hash: Array[Byte], view: BaseAccountStateView): Unit =
+    view.updateAccountStorage(contractAddress, hash, Array.emptyByteArray)
```

Comment:
let's prepend a prefix to the key hash, to avoid collisions with the message hash stored in line 125